net/zabbix4/Portfile

```diff
@@ -55,30 +57,40 @@ if {$zver == 42} {
 }
 
 if {$zver == 44} {
+    master_sites \
+        https://cdn.zabbix.com/zabbix/sources/oldstable/${zbranch}
     livecheck.regex     "zabbix-(4\.4\.\[0-9\]+).tar.gz"
     checksums \
-        rmd160  f2814c77fa6703bb80f25f891b9832570c63079b \
-        sha256  c0562245c75fc86c2c22d9a8e521c147dea832056b869d27488ea6130a253651 \
-        size    19457802
+        rmd160  f7c28344267a6646662c7e5bc678f22e1bf028ac \
+        sha256  2de09bd41ccb2534a0e12ca45c0e7da3896d95f0c4f9f7815477123a3d692c8f \
+        size    19560911
+}
+
+if {$zver == 5} {
+    livecheck.regex     "zabbix-(5\.\[0-9\.\]+).tar.gz"
```

Comment:
This and the other livecheck.regexes aren't escaping their periods properly. Correct escaping here would be:
```
    livecheck.regex     {zabbix-(5\.[0-9.]+)\.tar\.gz}
```